From: Ben Hutchings <ben@decadent.org.uk>
Subject: if_packet: Avoid ABI change in 2.6.32.43

A change in 2.6.32.43 replaces 2 bytes of automatic padding for
alignment with a named field, in 2 structures.  The changes are
binary-compatible, so just hide them from genksysms.

--- a/include/linux/if_packet.h
+++ b/include/linux/if_packet.h
@@ -63,7 +63,9 @@
 	__u16		tp_mac;
 	__u16		tp_net;
 	__u16		tp_vlan_tci;
+#ifndef __GENKSYMS__
 	__u16		tp_padding;
+#endif
 };
 
 /* Rx ring - header status */
@@ -104,7 +106,9 @@
 	__u32		tp_sec;
 	__u32		tp_nsec;
 	__u16		tp_vlan_tci;
+#ifndef __GENKSYMS__
 	__u16		tp_padding;
+#endif
 };
 
 #define TPACKET2_HDRLEN		(TPACKET_ALIGN(sizeof(struct tpacket2_hdr)) + sizeof(struct sockaddr_ll))
